index.html

```diff
@@ -20,143 +19,154 @@
+                    ><img src="./images/twitter.svg" alt="" class="icon-img"
+                  /></a>
+                </li>
+                <li>
+                  <a href="https://www.linkedin.com/in/blessing-asuquo-4509981a2/"
+                    ><img src="./images/linkedin.png" alt="" class="icon-img"
+                  /></a>
+                </li>
+                <li>
+                  <a href="https://medium.com/@blessingmike334"
+                    ><img src="./images/medium.svg" alt="" class="icon-img"
+                  /></a>
+                </li>
+                <li>
+                  <a href="https://github.com/blessing-michael"
+                    ><img src="./images/github.svg" alt="" class="icon-img"
+                  /></a>
+                </li>
+                <li>
+                  <a href="https://github.com/blessing-michael"
+                    ><img src="./images/Vector (3).svg" alt="" class="icon-img"
+                  /></a>
+                </li>
+              </ul>
+            </div>
+          </article>
+        </section>
+  
+        <section class="work-wrap">
+          <section class="work">
+            <div class="img">
+              <img src="./images/work-1.png" alt="work image" class="work-img" />
+            </div>
+  
+            <div class="works-info">
+              <h2 class="work-title">tonic</h2>
+  
+              <div class="works-details">
+                <h4 class="work-canopy">canopy</h4>
                 <p class="profession">
                   <span class="prof-span">
                     <img src="./images/Counter.png" alt="counter" /></span
                   >back end dev
                 </p>
                 <p class="profession">
                   <span class="prof-span">
-                    <img src="./images/Counter.png" alt="" /></span
+                    <img src="./images/Counter.png" alt="counter" /></span
                   >2015
                 </p>
               </div>
-              <div class="info-2">
-                <p class="info-p">
-                  A daily selection of privately <br />
-                  personalized reads; no accounts or sign-ups required.
-                </p>
-              </div>
-
-              <div class="ul">
-                <ul class="tech-stack">
-                  <li class="tech-name">HTML</li>
-                  <li class="tech-name">CSS</li>
-                  <li class="tech-name">Javascript</li>
-                </ul>
+  
+              <p class="work-info">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+  
+              <div class="tech-highligt">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
               </div>
-
-              <button class="project-btn">See project</button>
+              <button class="tech-project-btn" type="submit">See project</button>
```

Comment:
- [ ] [Optional] I would suggest you please implement the active, hover and pressed effect on you buttons it make good user experience. Please implement with all the buttons
### Design
![his](https://github.com/blessing-michael/portfolio/assets/81727988/9eb0038e-e750-452c-9bc8-34b91935316d)